test/Microsoft.ComponentDetection.VerificationTests/resources/pub/pubspec.lock

```diff
@@ -0,0 +1,35 @@
+# Generated by pub
+# See https://dart.dev/tools/pub/glossary#lockfile
+packages:
+  analyzer:
+    dependency: "direct dev"
```

Comment:
any chance we can use this to mark dependencies as dev dependencies? looks like a graph isnt possible from just this alone.

Reply:
Hi, no. This means that the developer added the package directly. 
If we want to look at packages that are relevant only for dev time (like test packages), they will be under 'dev_dependencies' and not under 'packages.'